{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "6580bdc3-4108-4925-8c75-0245ebdcd609-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "url", "value": "'/api/v1/users'"}, {"name": "body", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{\"code\":500,\"msg\":\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1605177272565, "stop": 1605177272575, "uuid": "c4d2dde3-50e3-43a4-871f-47137997bcae", "historyId": "c924d1af9f5731140632195fad922bd1", "testCaseId": "dc3f5c27b8f716b793706de44b0c3c3f", "fullName": "test_UserManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_UserManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "15468-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_UserManage"}]}